lib/lockstep_sdk/models/data_models.rb

```diff
@@ -21,110 +21,110 @@ module LockstepSdk
     # the priority of the task, and any amounts collected, paid, or credited for the task.
     class activity_model
```

Comment:
Camel Casing for class Names is required

Reply:
As per discussion on Slack, we're switching to ProperCase (aka PascalCase)